infrastructures/docker/src/test/java/org/eclipse/che/workspace/infrastructure/docker/local/projects/LocalProjectsFolderPathProviderTest.java

```diff
@@ -8,18 +8,19 @@
  * Contributors:
  *   Red Hat, Inc. - initial API and implementation
  */
-package org.eclipse.che.workspace.infrastructure.docker;
+package org.eclipse.che.workspace.infrastructure.docker.local.projects;
 
 import org.mockito.testng.MockitoTestNGListener;
 import org.testng.annotations.Listeners;
 
 @Listeners(value = {MockitoTestNGListener.class})
-public class LocalWorkspaceFolderPathProviderTest {
+public class LocalProjectsFolderPathProviderTest {
+  //TODO Fix tests
```

Comment:
Are you going to fix tests in this PR or another one?

Reply:
Yeah. See PR description

Reply:
Sorry, I missed WIP mark.